### DISCUSSION TITLE
Possibilidade de incluir husky

### DISCUSSION THREAD

**Discussion by LuccasTraumer** [techexpertspro/portal-ascepa#17] [Ideas]
Discutir possibilidade de incluir husky para Qualidade de Codigo do projeto.
Pensando em garantia de qualidade no que estamos desenvolvendo pensar em utilizar a Husky para isto.

**Comment by andrewarosario** [comment]
Criada Issue e vinculada no board #22

**Comment by FelipeDuarteLuna** [comment]
Bom dia Galera,

Nesse cenário que já ouve a discussão, foi criado a issue, issue está para ser concluida. Encerramos essa discussions ?

**Reply by andrewarosario** [reply]
Boa! Pull request já foi mergeado então vou fechar

**Comment by andrewarosario** [comment]
Husky implementado em #22
